### DISCUSSION TITLE
Rendering Images from /images directory

### DISCUSSION THREAD

**Discussion by sivadotblog** [McShelby/hugo-theme-relearn#654] [Q&A] [answered]
I have been using this theme for over an year now but i never upgraded until recently.  I see a new directory called /images and thats being rendered in the screenshots/index.en.md using markdown instead of hugo image syntax. but I couldn't get that working. I did update my submodule to the latest commit - so what am I missing. Appreciate your help. 

My site is in a private repo.
Hugo Version is .101

**Comment by McShelby** [comment, marked as answer]
From `config.toml`

````toml
# mounts are only needed in this showcase to access the publicly available screenshots;
# remove this section if you don't need further mounts
[module]
  [[module.mounts]]
````

But anyways I am wondering why you try to build the `exampleSite`. This is not recommended.

**Reply by sivadotblog** [reply]
Thanks @McShelby . That worked.
 I am not doing a build on the exampleSite. I just happen to come across and and I liked it better than the hugo syntax. Traditionally i have been using the hugo syntax
